app/translations/en.json

```diff
@@ -432,7 +432,10 @@
     "tapcopy-message": "Copied Key to Clipboard",
     "title": "Key",
     "address": "Address",
-    "text-readonly": "This is your viewing key. Please write/copy it down carefully. It is the only way to restore your actual wallet."
+    "text-readonly": "This is your viewing key. Please write/copy it down carefully. It is the only way to restore your actual wallet.",
```

Comment:
The viewing key doesn't restore spend authority.   So maybe we should change the second sentence to:

`Anyone who has a copy of the viewing key can observe all transactions to and from the associated wallet.`